docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -41,16 +41,21 @@
   content: |-
     We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is an image that we build from source code in a repository. An application is a collection of these components working together.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - Before you expand all of your configuration options, check the following:
+        - Is the repository correct? Click the GitHub link to check your repository URL or click the repository name above the GitHub link to edit it. When you're satisfied, click the checkmark. To exit editing mode, click the X.
```

Comment:
As a note here, for them to have gotten to the review/configure step, the URL they provided during Add Component must have been valid. If they typo'd to something that doesn't exist, they'll fail the validation step on the GitHub URL field. They could, however, be pointing to something they didn't intend to, so I guess clicking the link to see where it takes them could help realize if they copied the wrong thing.

The middle sentence reads as optional to (or click to edit it), but that editing context feels lost to me for the next two sentences which are only relevant if they did click to start editing the name. They can't see that check or X normally. Today, they have to click the pencil icon to edit the name, not the field itself. That might change as part of the redesign.

Reply:
good points, @mattreid. i also need to go back and make changes to the Add components Learn more.

re the lines that mention the checkmark and X, i tweaked. take a look?

yeah, I talked w/Missy and Christian. i am prepared for the design changing before Summit. i'm staying as closely in contact w/Missy as possible to stay on top of this. thank you!